mono/metadata/cominterop.c

```diff
@@ -459,7 +457,7 @@ cominterop_get_com_slot_for_method (MonoMethod* method, MonoError* error)
 		int offset = 0;
 		int i = 0;
 		ic = cominterop_get_method_interface (method);
-		if (!ic || !MONO_CLASS_IS_INTERFACE_INTERNAL (ic)) {
+		if (!ic) {
```

Comment:
I don't understand this change. Why is it ok to continue with `ic` that isn't an interface.  Wouldn't `mono_class_interface_offset` return -1 and then we'd crash on the `g_assert (offset >= 0)` ?